kafka/coordinator/consumer.py

```diff
@@ -233,7 +234,7 @@ def _on_join_complete(self, generation, member_id, protocol,
 
         # give the assignor a chance to update internal state
         # based on the received assignment
-        assignor.on_assignment(assignment)
+        assignor.on_assignment(assignment, generation)
```

Comment:
this will also crash if the user has configured a custom assignor that doesn't take the additional parameter